CMakeLists.txt

```
@@ -34,14 +34,16 @@ option(NEOFOAM_ENABLE_CPP_CHECK "Enable cpp check static analyzer" FALSE)
 option(NEOFOAM_ENABLE_CLANG_TIDY "Enable clang tidy static analyzer" FALSE)
 option(NEOFOAM_ENABLE_PCH "Enable precompiled header" FALSE)
 
+option(CPM_USE_LOCAL_PACKAGES "Try `find_package` before downloading dependencies" ON)
```

This is an option for CPM, so I don't think it should be presented as an option for neofoam. What I mean by that is that we should have instead:
```suggestion
if(NOT DEFINED CPM_USE_LOCAL_PACKAGES)
    message(STATUS "Set CPM_USE_LOCAL_PACKAGES=ON by default.")
    set(CPM_USE_LOCAL_PACKAGES ON)
endif()
```